lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.spec.ts

```diff
@@ -61,7 +61,25 @@ describe('CardViewDateItemComponent', () => {
 
     afterEach(() => fixture.destroy());
 
-    it('should pick date format from appConfigService', () => {
+    const updateDateTime = (key: string): Date => {
+        component.property.key = key;
+        component.dateFormat = 'M/d/yy, h:mm a';
+        component.property.value = new Date('Jul 10 2017 00:01:00');
+        const expectedDate = new Date('Jul 10 2018');
+        fixture.detectChanges();
+
+        const element = fixture.debugElement.nativeElement.querySelector(`span[data-automation-id="card-date-value-${key}"]`);
+        expect(element).toBeDefined();
+        expect(element.innerText).toEqual('Jul 10, 2017');
+        component.onDateChanged({ value: expectedDate });
+
+        fixture.detectChanges();
+        return expectedDate;
+    };
+
+    //we will enable this test when will change from moment to date-fns throughout the application
```

Comment:
I'm wondering if we should raise jira for that or maybe correct it for now based on actual functionality instead of disabling that so then when functionality will be corrected then test can be corrected too. @MichalKinas what do you think?

Reply:
We can keep this test case with xit as of now. We can raise a new Jira ticket to address this test case once we replace moment.js in whole application with date-fns. What do you think?

Reply:
I think it would be best not to create a partial PR for such a change. I believe the best approach would be to go through the whole ADF and replace it everywhere and then adjust all affected tests at once instead of disabling some of them. This approach makes it easier to miss something during testing phase, during unit test and e2e fixes and most importantly such a change shouldn't be merged partially as it may be released without being complete. cc @AleksanderSklorz @MateuszHY

Reply:
I agree with you @MichalKinas

Reply:
Agreed with @MichalKinas 
@jatin2008 @rbahirsheth what do you think ?

Reply:
Hi, @MichalKinas I do agree with you. We found some occurrences in the codebase but we could not find out the UI. We will analyze it. But, This PR is also part of the calendar replacement.